docs/metrics/README.md

```diff
@@ -32,6 +32,7 @@ MyFruitCounter.Add(7, new("color", "red"), new("name", "apple"), new("taste", "s
   [`TagList`](https://learn.microsoft.com/dotnet/api/system.diagnostics.taglist?view=net-7.0#remarks)
   avoids allocating any memory for up to eight tags, thereby, reducing the
   pressure on GC to free up memory.
```

Comment:
```suggestion
  pressure on GC to free up memory. DO NOT use `TagList` when emitting less than three tags as that would lead to poorer performance than individually emitting the tags.
```

Reply:
`<3' (less)  != '<=3' (up to). Based on https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1335#discussion_r1309483604 '<=3' should be used.

Reply:
> `<3` (less) != `<=3` (up to). Based on [open-telemetry/opentelemetry-dotnet-contrib#1335 (comment)](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1335#discussion_r1309483604) `<=3` should be used.

Sure.
Fixed.